packages/prisma/migrations/20230524105015_added_newbooker_feature_flag/migration.sql

```diff
@@ -0,0 +1,9 @@
+INSERT INTO
```

Comment:
Migration for the feature flag. I created this manually, I think that's what we're supposed to do, right?

Reply:
Yeah this seems to be correct.  

run 
`prisma migrate dev --create-only` 
add the migration in to the created file

Reply:
It is correct. Prisma isn't involved here.